qatrack/qa/control_chart/control_chart.py

```
@@ -417,11 +417,11 @@ def get_subgroups(x, sgSize, dates):
 ################################################################################
 
 def get_ranges(sg, n):
-
+    print("sg = %r" % sg, "n = %r" % n)
```

Ruff (0.15.14)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)